### ISSUE TITLE
Assertion `usb_packet_is_inflight(p)' failed.

### ISSUE THREAD

**Issue by asdfugil** [TrungNguyen1909/qemu-t8030#50]
The Linux VM crashed while telneting into iOS, with message 
```
qemu-system-x86_64: ../hw/usb/core.c:523: void usb_cancel_packet(USBPacket *): Assertion `usb_packet_is_inflight(p)' failed
```

The most reliable way to produce the same error message would be trying to upload a `iBSS.n71.RELEASE.im4p.dec` to the s8000 securerom VM, but it can happen in T8030 VMs too (such as the telnet crash).

**To reproduce**
Method 1: upload *anything* (such as a decrypted iBSS, or 1MB of random data) to S8000 securerom
```
irecovery -f iBSS.n71.RELEASE.im4p.dec
```
Method 2: Setup telnetd on the iOS side, telnet into it in the Linux VM, and do some random stuff. It will crash

**Result**
Linux VM crashed.

No core dumps are generated.
Host is debian 11 bullseye
qemu-t8030 commit: ba738a11f8750fbc9e7224f5b34628d4fe282991

**Comment by asdfugil** [comment]
In particular I would also like to boot S8000 kernels but it seems unsupported for the time being...

**Comment by TrungNguyen1909** [comment]
Should be fixed in 2f0f0f7

@asdfugil, why would you want to boot S8000 kernels though? It would not be any better than the current state of T8030 kernels (No nvme even). I guess Aleph Security's fork is better for that purpose.
